common/api/imodeljs-frontend.api.md

```diff
@@ -12790,6 +12795,8 @@ export abstract class ViewState3d extends ViewState {
     getCartographicHeight(point: XYAndZ): number | undefined;
     getDisplayStyle3d(): DisplayStyle3dState;
     // (undocumented)
+    getEarthFocalPoint(): Point3d | undefined;

```

Comment:
Can you document all the new public undocumented APIs you've added?

Reply:
Done...  (This one had a missing ** in the comment).